integrations/client/test_delphi_epidata.py

```diff
@@ -54,12 +53,12 @@ def test_covidcast(self):
 
     # insert placeholder data: three issues of one signal, one issue of another
     rows = [
-      self._make_placeholder_row(issue=self.DEFAULT_ISSUE + i, value=i, lag=i)[0]
+      CovidcastRow(issue=20200202 + i, value=i, lag=i)
```

Comment:
What's the benefit of switching to constant 20200202 here?

Reply:
Faster to read code when the variables stay local and the abstraction seemed to have no other benefit, especially when I deleted _make_placeholder_row in CovidcastBase (since CovidcastRow already has defaults), it meant I needed to look into CovidcastBase just to find an arbitrary value.